web/profiles/joinup/joinup.behat.inc

```diff
@@ -475,4 +475,26 @@ class JoinupSubContext extends DrupalSubContextBase implements DrupalSubContextI
     $this->getSession()->getPage()->find('named', ['fieldset', $fieldset])->pressButton($button);
   }
 
+  /**
+   * Fills a date or time field at a datetime widget.
+   *
+   * Example: When I fill in "Date" with "29-08-2016" at "Start date".
+   * Example: When I fill in "Time" with "26:59:00" at "Start date".
```

Comment:
This sounds a bit weird. I talked also with Pieter and he made a suggestion of 'When I enter the time "26:59:00" for the "Start date" field'.
Mine would be "When i fill in "Start date" with the date/time xxx"